packages/ember-glimmer/tests/integration/components/curly-components-test.js

```diff
@@ -2101,4 +2101,40 @@ moduleFor('Components test: curly components', class extends RenderingTest {
 
     this.assertText('initial value');
   }
+
+  ['@htmlbars a two way binding flows upstream when set during init (#11192)']() {
```

Comment:
maybe a better nomenclature is external attributes vs internal attributes. Rather then describing this as upstream or downstream binding propagation.